salt/_logging/impl.py

```diff
@@ -419,6 +426,7 @@ def makeRecord(
     logging.addLevelName(PROFILE, "PROFILE")
     logging.addLevelName(TRACE, "TRACE")
     logging.addLevelName(GARBAGE, "GARBAGE")
+    logging.addLevelName(DEBUG, "DEBUG")
```

Comment:
The DEBUG level isn't present?!
`addLevelName` doesn't complain if there's already a level name by the same name?

Reply:
Hmm, we've been running this patch for a while and it doesn't complain, but I'll double check if this is strictly necessary on the context of the deadlock.